src/measure_theory/measure/vector_measure.lean

```diff
@@ -465,6 +465,27 @@ end measure
 
 namespace vector_measure
 
+open measure
+
+/-- A vector measure over `ℝ≥0∞` is a measure. -/
+def ennreal_to_measure (v : vector_measure α ℝ≥0∞) : measure α :=
```

Comment:
@sgouezel Should I make `measurable_space \a` an implicit argument here? In fact, should I do that for this entire file?

Reply:
In most situations yes. But not always, for instance `equiv_measure` requires the `measurable_space` structure a priori. I'd say, try to put it implicit in the whole file, and see what breaks.

Reply:
Everything seems to work except for `vector_measure.map` and `vector_measure.restrict`